spec/hydra/works/services/persist_derivatives_spec.rb

```diff
@@ -88,5 +88,21 @@
         expect(generic_file.thumbnail.mime_type).to eq('image/jpeg')
       end
     end
+
+    context 'with a pdf file' do
```

Comment:
I don't understand how this fits with the rest of this pull request. Was this supposed to go here?